### PR TITLE
added new maps

### PR DESCRIPTION
**Pull request by grad-ler**
added missing maps and reordered old ones

### CODE REVIEW

File: cfgGradIslands.hpp [hunk 1]
```diff
@@ -815,4 +815,130 @@ class cfgGradIslands {
...
+    class uzbin {
+        displayName = "Uzbin Valley";
+        type = "middle eastern";
+        isWoodland = false;
+        spawnPosBlu[] = {256.679,995.629,0};
+        spawnPosOpf[] = {9451.65,9972.54,0};
+        spawnPosInd[] = {10447.4,1511.11,0};
+    };
+
+    class uzbin {
```
Comment (diwako) [inline cfgGradIslands.hpp]: Duplicate entry

**Review by McDiod — APPROVED** [review]

**Comment by Fusselwurm** [comment]
it would be neat if the configs were sorted alphanumerically too.

**Review by Fusselwurm — APPROVED** [review]

File: cfgGradIslands.hpp (hunk 1 above)
Comment (grad-ler) [reply cfgGradIslands.hpp]: fixed with 682954c

**Comment by grad-ler** [comment]
> it would be neat if the configs were sorted alphanumerically too.

adjusted in da7437d

**Review by Fusselwurm — APPROVED** [review]
to94kbbzmv7vmpwe9zfe

**Review by TheWillard — APPROVED** [review]

**Pull request merged**